pkgs/development/python-modules/python-frontmatter/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, lib, pythonPackages, buildPythonPackage, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "python-frontmatter";
+  version = "0.5.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "09n44qmzbzpgddgzmissaawa671k1w29j8mqv1l0r7wfqc7ykhm9";
+  };
+
+  propagatedBuildInputs = with pythonPackages; [
+    pyyaml
+    six
+  ];
+
+  doCheck = false;
```

Comment:
There are no tests included. Please try to checkout from source and check if they have unit tests, and try to run them. Unit tests give a good indication that they package has a high degree of validity and correctness given the python package set.

If tests are not available, then please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors. Please see [pythonImportsCheck documentation](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonimportscheck) for more information.